OIPS/oip-draft_gateway_composer.md

```diff
@@ -40,63 +40,66 @@ interaction flows across multiple contracts.
 ## Specification
 <!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations.-->
 
-We can think of the UX composer as a macro automating user actions. Users can
-remove their macros and deploy improved ones.
+We can think of the Gateway composer as a macro automating user actions. Users
+can remove their macros and deploy improved ones.
 
-We detail a UX composer for a user who wants to stake OST
+We detail a Gateway composer for a user who wants to stake OST
 in a Branded Token (BT) contract and following that stake the Branded Tokens
 into a gateway contract to mint the same amount as Utility Branded Tokens (UBT)
 on the sidechain to use within the application of the token.
 
-Every staker deploys her own Gateway composer contract (UXC),
+Every staker deploys her own Gateway composer contract,
 because the messagebus nonce in gateway locks a single message per staker -
-and the UXC contract address is the `staker` for the gateway contract.
-A UXC can be (re-)used for multiple gateways in parallel.
+and the Gateway contract address is the `staker` for the gateway contract.
```

Comment:
@abhayks1 small error from replacing; "... and the Gateway Composer (GC) address is the `staker` for the gateway contract."

Reply:
Done!
Also updated case of "Gateway composer" to "Gateway Composer" in some places.